src/app/services/ws.service.ts

```diff
@@ -164,11 +186,7 @@ export class WebSocketService {
   }
 
   private subscribeToJobUpdates(jobId: number): Observable<Job> {
-    return this.subscribe('core.get_jobs').pipe(
-      filter((apiEvent) => apiEvent.id === jobId),
-      map((apiEvent) => apiEvent.fields),
-      takeUntil(this.clearSubscriptions$),
-    );
+    return this.store$.select(selectJob(jobId)).pipe(takeUntil(this.clearSubscriptions$));
```

Comment:
This doesn't look right. WebsocketService is at lower level conceptually, it shouldn't know anything about Store.

Reply:
Yeah, agreed. This is unrelated change.